compiler/base/Cargo.toml

```diff
@@ -3,6 +3,12 @@ name = "playground"
 version = "0.0.1"
 authors = ["The Rust Playground"]
 
+[profile.dev]
```

Comment:
This file is actually [generated by the `top-crates` script](https://github.com/integer32llc/rust-playground/blob/master/top-crates/src/main.rs#L42-L47) as part of the calculation of top N downloads and dependencies.

We will want to move this change to the creation step so it doesn't get lost every time the crates are updated.